joomla/joomla-cms
# [4.1] fix parks image

Pull Request for Issue # .

### Summary of Changes

Parks image doesn't display properly due to improper relative path.

### Testing Instructions
Install sample testing data.
Click on "parks" on the menu at the top
Notice that the image does not display.
Verify the url for the image is "http://j4.local/index.php/images/sampledata/parks/banner_cradle.jpg" ( your uri will be different of course )
view that the image displays.
verify url for the image is "http://j4.local/images/sampledata/parks/banner_cradle.jpg"



### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/165410355-3563c2f4-4029-4ff8-9de6-7a1e907633ed.png)



### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/165410378-0cf785aa-8c08-4cdc-9a2b-39cefdf67085.png)



### Documentation Changes Required

None
